skel/manifests/example-pod.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: alpine
+  name: example-pod
 spec:
   restartPolicy: Never
   containers:
-  - name: alpine
+  - name: example
   image: "alpine:3.2"
```

Comment:
I don't know if that's a terribly big deal, but it certainly wouldn't hurt. That said... could you _do_ "scratch" and a random command? I don't think scratch has any commands? Maybe "alpine:3.2" with something like "sleep 900" or something?